src/map/zone_entities.h

```diff
@@ -71,6 +72,7 @@ class CZoneEntities
     bool			CharListEmpty();
     uint16			GetNewTargID();
 
+    EntityList_t	m_allyList;
```

Comment:
Ironically I had done that at first, then changed it. I was thinking that it may be useful in more cases outside of instances eventually - like besieged or campaign for example. But then again, my targeting system is tied to the other two. *sigh* refactoring needed either way. Thoughts on that?